README.md

```diff
@@ -1,5 +1,10 @@
 Set up Dwelling Flask Testing Backend (for the first time)
-NOTE: no database setup here. Backend uses list-based live data 
+NOTE: Database is SQLite3 via SQLAlchemy 
+
++ Github Repo: {https://github.com/codeforpdx/dwellinglybackend}
++ Live Server: {https://dwellinglyapi.herokuapp.com/ } 
+
+### To Start Server
 
 1. Fork the backend and clone a copy ( https://help.github.com/en/github/getting-started-with-github/fork-a-repo )
 2. Install Python ( https://realpython.com/installing-python/ )
```

Comment:
For easier readability, we can put the commands in the steps below in a `code` format, and do a hard new line since it doesn't seem like the new lines are being respected

Reply:
I've made all the readability changes in a local branch of the repo. 

How can I provide the updated file contents?